homeworks/elena-savitskaya/homework-7/firstScript.js

```diff
@@ -0,0 +1,41 @@
+// Вариант 1
+
+function Store(name) {
+    this.name = name;
+    this.orders = [1, 2, 3, 4, 5, 6];
+
+    this.addOrder = function addNumberToOrder(number) {
```

Comment:
```suggestion
    this.addOrder = function addOrder(number) {
```
Better to have the same name

OR

anonymous function
```suggestion
    this.addOrder = function(number) {
```